### DISCUSSION TITLE
How to use swup.loadPage

### DISCUSSION THREAD

**Discussion by mocodev** [swup/swup#626] [Help] [answered]
I'm building a kiosk using SWUP. If a user is idle on an interior page we have a timeout function that runs to return the user to the home screen. I cannot figure out how to use `swup.loadPage` though, it always results in a `swup.loadPage is not a function` error.

```
function returnHome () {
    swup.loadPage({url: '/index'}); 
}
```

**Comment by mocodev** [comment, marked as answer]
Turns out I had a javascript scope issue.

**Reply by hirasso** [reply]
Great that you solved it and thanks for reporting back!
